geomesa-convert/README.md

```diff
@@ -260,12 +261,20 @@ Example: ```dateHourMinuteSecondMillis('2015-01-01T00:00:00.000')```
 
 #### millisToDate
 
-Description: Create a new date from as long representing millis since 1970.
+Description: Create a new date from a long representing milliseconds since January 1, 1970.
 
 Usage: ```millisToDate($1)```
 
 Example: ```millisToDate('1449675054462'::long)``` 
 
+#### secsToDate
+
+Description: Create a new date from an integer (NOT long) representing seconds since January 1, 1970.
```

Comment:
I remember now.  There was an issue that reading in a *.conf file (I forget which version) didn't have a ::long cast option, so I had to use int.  Fortunately, int is good until 2038.